libs/wingsdk/src/target-awscdk/function.ts

```diff
@@ -36,6 +36,13 @@ export class Function extends cloud.Function implements IAwsFunction {
     // bundled code is guaranteed to be in a fresh directory
     const bundle = createBundle(this.entrypoint);
 
+    const logRetentionDays =
+      props.logRetentionDays === undefined
+        ? 30
+        : props.logRetentionDays < 0
+        ? undefined // Negative value means Infinite retention
```

Comment:
Shouldn't we allow this as well? (what if a wing user does want infinite retention?)

Reply:
Yes, it should allow infinite retention.
If a user specifies a negative value (e.g. `-1`), `logRetentionDays` will be `undefined`, which means infinite retention in CloudWatch.

Reply:
oh I see now, nice!